### ISSUE TITLE
Update install.sh for Void Linux

### ISSUE THREAD

**Issue by neilotoole** [neilotoole/sq#316]
**Is your feature request related to a problem? Please describe.**

**Describe the solution you'd like**
`sq` is now included in the [Void Linux](https://voidlinux.org) package repo (see [PR](https://github.com/void-linux/void-packages/pull/45023)). The `sq` [install script](https://github.com/neilotoole/sq/blob/master/install.sh) should be updated to detect Void, and use the Void [`xbps`](https://github.com/void-linux/void-packages) package manager.

**Comment by neilotoole** [comment]
@icp1994 I've added a call to `xbps-install` to the generic [`install.sh`](https://github.com/neilotoole/sq/blob/master/install.sh#L28) script. So, this should now work on Void systems:

```shell
/bin/sh -c "$(curl -fsSL https://sq.io/install.sh)"
```

Any chance you could verify that on your end? Any other feedback welcome too!

**Comment by icp1994** [comment]
Yes, both the install script and the ack works fine for me as is.
